### ISSUE TITLE
Background image not rendering on lower resolution mobile

### ISSUE THREAD

**Issue by mjashanks** [aletheia-foundation/aletheia-foundation.github.io#12]
Background image is not displaying on lower resolution mobiles. 

This is the case on my Samsung Galaxy S5, but not on a recent iPhone (not sure of versions yet)

Possibly a useful article: https://stackoverflow.com/questions/23586530/css-background-image-not-displaying-on-mobile

**Comment by rowhitswami** [comment]
I'd like to take this up.

**Comment by rowhitswami** [comment]
@mjashanks Which image you're talking about?

**Comment by r3versi** [comment]
/images/bg_100pc.jpg

**Comment by rowhitswami** [comment]
May I know the resolution of the device in which you're facing the problem?

**Comment by mjashanks** [comment]
@rowhitswami  thanks... Phone is a Samsung Galaxy S5 Mini - Resolution 720 x 1280

**Comment by rowhitswami** [comment]
This is what I am getting on your said resolution. The only problem I can see is the image is repeated at the bottom.

<img src="https://image.ibb.co/kYBa86/Screenshot_from_2017_10_16_21_52_55.png" alt="Screenshot_from_2017_10_16_21_52_55" border="0">

**Comment by mjashanks** [comment]
Yes... i see the same if i set my screen size to those dimensions.. it is only on my phone.

Possibly "DPI" is the issue, rather than resolution (?)

**Comment by mjashanks** [comment]
After seeing your image though, i realised that the banner takes up alot more room... so i measured my screen size directly. Its telling me that it is 640 x 360

Sorry, dont know why my phone is not operating at the resolution its supposed to!

**Comment by mjashanks** [comment]
![screenshot_2017-10-16-18-34-48](https://user-images.githubusercontent.com/3524181/31630866-d3bb96c0-b2af-11e7-8683-085db5e1505d.png)

This is how it looks on my phone. I'm guessing that this is not a huge issue, as apparently my phone has a very low resolution. Even at that, the site is still usable.

I will leave the issue open, but I think that it is fairly low priority
